portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -104,11 +118,14 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
         // Get the input from the form and log time of comment
         String commentText = request.getParameter(COMMENT_FORM_PARAMETER);
         long timestamp = System.currentTimeMillis();
+        String imageURL = getUploadedFileUrl(request, "image");
```

Comment:
Since "image" is hardcoded here, can you directly use that in the `getUploadedFileUrl` method instead of passing it as an argument? Also worth making `"image"` a `private static final` class level variable.

Reply:
done